docs/as-a-resource/from-data-to-array.md

```diff
@@ -30,7 +15,11 @@ Or transform a data object to an array:
 SongData::from(Song::first())->toArray();
 ```
 
-By default, calling `toArray` on a data object will transform all properties to an array. This means that nested data objects and collections of data objects  will also be transformed to arrays. Other complex types like `Carbon`, `DateTime`, `Enums`, ... will be transformed into a string. We'll see in the [transformers](/docs/laravel-data/v4/as-a-resource/transformers) section how to configure and customize this behavior.
```

Comment:
Let's keep these on one line, the spate docs site will make sure they're readable